ait/dsn/plugins/TCP.py

```diff
@@ -238,14 +239,23 @@ def recv_as_client(self):
         if rx:
             try:
                 data = self.socket.recv(self.receive_size_bytes)
-                msg = (f"{self.log_name} Receiving {self.topic} "
-                       f"subscription from {self.hostname} {data}")
-                log.debug(msg)
-                return data
+                if not data:
```

Comment:
I think the docstring for this function might be off, the name is "recv_as_client" but docstrings says "receive data as a server"